app/src/docs/v1.api-spec.yaml

```diff
@@ -17,30 +17,348 @@ security:
-        - Hello
+        - Object
+      parameters:
+        - $ref: '#/components/parameters/PathMessageId'
       responses:
         '200':
-          description: Hello result
+          description: Returns the object information
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/HelloDetail'
+                $ref: '#/components/schemas/ObjectDetails'
         '401':
           $ref: '#/components/responses/UnauthorizedError'
         '403':
           $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '422':
+          $ref: '#/components/responses/UnprocessableEntity'
         default:
           description: Unexpected error
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+    post:
+      summary: Updates an Object
+      description: >-
+        This endpoint will update an object.
+      operationId: PostUpdateObject
+      tags:
+        - Object
+      parameters:
+        - $ref: '#/components/parameters/PathMessageId'
+      responses:
+        '201':
+          description: Returns udpate Object response.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MergeResponse'
+        '401':
+          $ref: '#/components/responses/UnauthorizedError'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '409':
+          $ref: '#/components/responses/Conflict'
+        '422':
+          $ref: '#/components/responses/UnprocessableEntity'
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+    delete:
+      summary: Deletes the object
+      description: >-
+        This endpoint will delete the Object specific to the objId
+      operationId: DeleteObject
+      tags:
+        - Object
+      parameters:
+        - $ref: '#/components/parameters/PathMessageId'
+      responses:
+        '202':
+          description: Request is acknowledged but not necessarily completed
+          headers:
+            Content-Location:
+              description: >-
+                URL location to affected resource. Use the same query and path
+                parameters as this request to find the affected resource.
+              example: /api/v1/status/00000000-0000-0000-0000-000000000000
+              schema:
+                type: string
+                format: uri
+        '401':
+          $ref: '#/components/responses/UnauthorizedError'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '409':
+          $ref: '#/components/responses/Conflict'
```

Comment:
Current deletion behavior is a 202 when it gets deleted - repeated calls will yield a 404 since it's no longer there. 409 won't be a planned thing.

Reply:
Should I be removing the 404 and 409 calls then from here?

Reply:
Just remove the 409 - we'll still have 202s and 404s on this endpoint I think.